### ISSUE TITLE
Availability? 

### ISSUE THREAD

**Issue by necrogami** [surrealdb/surrealdb.php#1]
I was wondering the ETA for when the PHP driver will be available for surrealdb?

**Comment by iDevelopThings** [comment]
I'm also curious... I'm tempted to start writing a lib to handle some of it... 
can someone at the team confirm if this has been started yet?
and if so, maybe push the base so the community can add to it 

I have some ideas on how to architect it using websockets, but it would require running a php background instance and some kind of IPC setup to communicate between the main instance and background instance(im not actually sure if it would work though, haven't fully thought through it, there's probably some blockers 😅 )

**Comment by ideiasfrescas** [comment]
@iDevelopThings do you intend to write an extension or php class? extension would be great as one would not have the need to add bloat to the codebase and it could work like mysqli does ( or other database extension ).

**Comment by leander-hass** [comment]
I would prefer a PHP-class/library, as I imagine developing, building and installing extensions is quite more work than a simple composer package. But yeah, I also thought about starting it.

**Comment by mcprog1** [comment]
I support the idea of ​​being able to start developing the library for PHP, but can you tell me what is left? Since there is nothing at all done, and if so, how can we make a request to make said library.

**Comment by Guikingone** [comment]
Hi 👋🏻 

Small question, could we help to build this library? 

Maybe steps by steps but is there anything we can do to bring this library to life? 🤔

**Comment by atapatel** [comment]
@Guikingone as a step by step i would say we can start with official composer package.

**Comment by naisofly** [comment]
@Guikingone @atapatel 100%
DevRel team is here to support you as needed.

**Comment by Guikingone** [comment]
Agree, starting an official composer package can help bootstrapping the client, I can start working on the websocket part if no one is currently working on it.

**Comment by admtech** [comment]
What is the current status? Now that version 1.0 is out, we're all excited about the PHP SDK. We would be pleased to receive an official composer package.

**Comment by Guikingone** [comment]
Hi @admtech 👋🏻 

Regarding the websocket client, I'm currently working on it, I don't know for the HTTP one but the websocket should be ready soon.

**Comment by ahmedali5530** [comment]
How to run live queries? or should i stick to js version?

**Comment by welpie21** [comment]
> How to run live queries? or should i stick to js version?

Hello, I'm the community maintainer of the current surrealdb.php SDK. Currently the php driver does not support live queries at the moment. I'm looking in the future to support it.

You can stick to the JS version for live queries. I will give you a heads-up when i'm beginning implementing the live query functionalities 😄 

If you have any questions feel free to ask here or tag me "Beau" - "gigapastor" on discord.

Have a great day 😄 

Best regards,

Beau

**Comment by kearfy** [comment]
Hey all, as this library has since been released, I'll be closing this issue now 👍
